### PR TITLE
fix: upgrade lodash to fix vulnerability

### PR DESCRIPTION
**Pull request by hkang1**
## Commentary (optional)

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234

### CODE REVIEW

File: webui/tests/package-lock.json [hunk 1]
```diff
@@ -1419,6 +1427,12 @@
           "integrity": "sha512-EykJT/Q1KjTWctppgIAgfSO0tKVuZUjhgMr17kqTumMl6Afv3EISleU7qZUzoXDFTAHTDC4NOoG/ZxU3EvlMPQ==",
           "dev": true
         },
+        "lodash": {
+          "version": "4.17.15",
```
Comment (hamidzr) [inline webui/tests/package-lock.json:1431]: this is  using a version less than 4.17.19
> │ Patched in    │ >=4.17.19                                                    │

so it shows up as a vulnerability. should we force resolution or update the responsible package?

**Comment by hamidzr** [comment]
these seem to be the ones need upgrading (ones with 15)
```
hrz ➜ tests git:(fix-lodash-vulnerability) ✗ npm ls lodash                                                                                                                                                                                                                                                                                                               
webui-e2e-test@1.0.0 /home/hmd/projects/da/determined/webui/tests
├─┬ @cypress/webpack-preprocessor@5.4.1
│ └── lodash@4.17.15 
├─┬ @typescript-eslint/parser@3.2.0
│ └─┬ @typescript-eslint/typescript-estree@3.2.0
│   └── lodash@4.17.19  deduped
├─┬ cypress@4.8.0
│ └── lodash@4.17.15 
└─┬ eslint@7.2.0
  ├─┬ inquirer@7.1.0
  │ └── lodash@4.17.19  deduped
  ├── lodash@4.17.19 
  └─┬ table@5.4.6
    └── lodash@4.17.19  deduped

```

File: webui/tests/package-lock.json (hunk 1 above)
Comment (hkang1) [reply webui/tests/package-lock.json:1431]: `"@cypress/webpack-preprocessor": "^5.4.1"` is the responsible package and they are still using `4.17.15`. using force resolution to resolve this

Comment (hamidzr) [reply webui/tests/package-lock.json]: I looked that up it seems they don't have a newer release

**Pull request merged**